handlers/main.yml

```diff
@@ -5,10 +5,10 @@
     state: reloaded
   when:
     - openhpc_slurm_service is not none
-    - openhpc_slurm_service_enabled | bool
```

Comment:
Actually if we're fixing all this can we name this "Reload SLURM service" please?? It doesn't actually restart it, which is pretty key!

Reply:
Agree that we should make that change, but I thought it would be best in another PR.